.travis.yml

```diff
@@ -1,7 +1,6 @@
 language: python
 dist: bionic
 python:
-- 3.5
```

Comment:
There's still the trove classifier in setup.py.  And maybe consider adding `python_requires=">=3.6"` to setup.py.

Reply:
thanks!